MonoGame.Framework/Graphics/Effect/SpriteEffect.cs

```diff
@@ -1,24 +1,13 @@
-#region File Description
-//-----------------------------------------------------------------------------
-// SpriteEffect.cs
-//
-// Microsoft XNA Community Game Platform
-// Copyright (C) Microsoft Corporation. All rights reserved.
-//-----------------------------------------------------------------------------
```

Comment:
Not sure yet.

This and the other stock effects came from one of the Microsoft samples.  So we want to leave MS's copyright there.

However we can probably clean it up... remove the `------` crap.  Remove the redundant `// SpriteEffect.cs`.

Not sure how much else can change.